app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/govuk_wrapper.scala.html

```diff
@@ -140,6 +141,9 @@
 
 @mainContentHeader = {
     @contentHeader.fold(Html(""))(header => uiMainContentHeader(contentHeader = header))
+    <div class="centered-content">
+        @languageSwitcher(appConfig.enableLanguageSwitching)
+    </div>
 }
```

Comment:
@nubz added in for now just to test - needs to be relocated as appropriate.

Reply:
I removed the unnecessary wrapping div (which has undesired effect of nesting centred content with unwanted padding inside an already centred parent)